model/atmosphere/dycore/tests/dycore_tests/test_solve_nonhydro.py

```diff
@@ -119,6 +119,8 @@ def test_nonhydro_predictor_step(
     ndyn_substeps,
     caplog,
 ):
+    if (backend is not None) and ("gtfn_cpu" in backend.name):
```

Comment:
@kotsaloscv Was this stencil that did not work with dace backend either? If this is the case, we should also exclude dace backends: `("gtfn" in backend.name or "dace" in backend.name)`

Reply:
it didn't work with `gtfn_cpu`

Reply:
I know, and I suspect that it would not even work with dace. I have asked @kotsaloscv for confirmation.

Reply:
this is the stencil that does not work with DaCe: `model/atmosphere/dycore/tests/dycore_stencil_tests/test_solve_tridiagonal_matrix_for_w_forward_sweep.py`
and I see that it is called from this nonhydro test as well.
So, DaCe does not work.

Reply:
I see, then it could be a different reason. @nfarabullini you can ignore my comment, we will deal with the dace failures in separate PR.